apps/api/src/chapter/chapter.controller.ts

```diff
@@ -20,25 +26,21 @@ export class ChapterController {
     private readonly adminChapterService: AdminChapterService,
   ) {}
 
-  // @Get("lesson")
-  // @Roles(...Object.values(USER_ROLES))
-  // @Validate({
-  //   request: [
-  //     { type: "query", name: "id", schema: UUIDSchema, required: true },
-  //     { type: "query", name: "courseId", schema: UUIDSchema, required: true },
-  //   ],
-  //   response: baseResponse(showLessonSchema),
-  // })
-  // async getLesson(
-  //   @Query("id") id: UUIDType,
-  //   @Query("courseId") courseId: UUIDType,
-  //   @CurrentUser("role") userRole: UserRole,
-  //   @CurrentUser("userId") userId: UUIDType,
-  // ): Promise<BaseResponse<ShowLessonResponse>> {
-  //   return new BaseResponse(
-  //     await this.lessonsService.getLesson(id, courseId, userId, userRole === USER_ROLES.admin),
-  //   );
-  // }
+  @Get()
+  @Roles(...Object.values(USER_ROLES))
+  @Validate({
+    request: [{ type: "query", name: "id", schema: UUIDSchema, required: true }],
+    response: baseResponse(showChapterSchema),
+  })
+  async getLesson(
```

Comment:
Maybe it should be getChapterWithLesson also, because as i see you are getting here chapter, not lesson